plugins/kafka/kafka.ecllib

```diff
@@ -212,9 +212,10 @@ EXPORT Consumer(VARSTRING topic, VARSTRING brokers = 'localhost', VARSTRING cons
         END;
 
         // Set the offset for each partition on each node
-        result := PROJECT
+        result := NORMALIZE
             (
-                distOffsets,
+                NOFOLD(distOffsets),
```

Comment:
That makes no sense. If the DISTRIBUTE can be optimized away, then why would you not want it to be? If if can't be optimized away but the compiler did so anyway, then that sounds like either a compiler bug (that should be investigated) or a misunderstanding on the part of the ECL programmer (that should be remedied).